src/channel/playlist.js

```diff
@@ -973,6 +973,10 @@ PlaylistModule.prototype._addItem = function (media, data, user, cb) {
         }
     }
 
+    if (media.meta.rating) {
```

Comment:
Probably best to check if rating is equal to `"ytAgeRestricted"` in case there are other ratings that are not blocked.

https://developers.google.com/youtube/v3/docs/videos#contentDetails.contentRating.ytRating

Reply:
Added this. Initially thought I'd leave it as is since the current API says that's the only value it can have, but they might add new stuff there.